src/Modules/Authentication/Login/Login.tsx

```diff
@@ -1,7 +1,156 @@
 import styles from './Login.module.css';
+import { IconButton, InputAdornment, Stack } from "@mui/material";
+import image from "../../../assets/images/loginImg.png";
+import Typography from "@mui/material/Typography";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import { Link, useNavigate } from "react-router-dom";
+import { useForm } from "react-hook-form"
+import { AUTH_URL, axiosInstanceAdminAuth } from '../../../Services/END_POINTS/ADMIN/URLS';
+import { useContext, useState } from 'react';
+import { AuthContext } from '../../../Context/AuthContext';
+import { toast } from 'react-toastify';
+import { EMAIL_VALIDATION, PASWORD_VALIDATION } from '../../../Services/Validation/VALIDATION';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import RemoveRedEyeIcon from '@mui/icons-material/RemoveRedEye';
+interface loginData {
+  email: string;
+  password: string;
+}
 
+interface LoginResponse {
+  token: string;
+  message: string;
+}
 export default function Login() {
-  return (
-    <div>Login</div>
-  )
+  const [isPasswordVisable, setIsPasswordVisable] = useState(false);
+  const navigate = useNavigate();
+  const { saveLoginData } = useContext(AuthContext)
```

Comment:
I suggest using the context using custom hook as it will make using it easier and we won't need to check for null or undefined values.
You can watch this video for this pattern.
https://youtu.be/I7dwJxGuGYQ?si=MoYCwG9hYmIT2_Au